source/assets/stylesheets/styles/_project_team_single.scss

```diff
@@ -0,0 +1,7 @@
+.project-team__headshot-container-single {
+  	display: block;
+  	float: left;
+  	position: relative;
+  	left: 50%;
+  	margin-left: -80px;
+}
```

Comment:
Please try to avoid brute forcing CSS like this - the desired result you are after is a single centered image which can be done simply using a block element with `margin: 0 auto;` or inline element with `text-align: center;`. When you find yourself producing CSS with negative margins, absolute positioning then you can probably solve yourself trouble by changing the HTML that you are targeting.

In our particular case, a single team member was producing undesired styling, mainly from what was inherited from the `project-team__headshot-container` class - which was originally designed for multiple images. See [my other comment](https://github.com/unboxed/ubxd_web_refresh/pull/190/files#r71673022) for my suggested approach.